### ISSUE TITLE
Support for docker

### ISSUE THREAD

**Issue by johnbradley101** [michaelrsweet/lprint#131]
HI,  is it possible to add support for Docker easily? 
this would be great to run on a docker container on a synology NAS to make network label printers accessible over airprint.

**Comment by michaelrsweet** [comment]
@johnbradley101 I'm not sure about USB support, but will investigate.

**Comment by johnbradley101** [comment]
> @johnbradley101 I'm not sure about USB support, but will investigate.

thanks - that would be great.  I've been able to share a label printer using the synology print server, so it is accessible from a windows PC / Mac over the network, but I've not been able to get it to work over airprint.  So would it even need to have the USB aspect supported if it is available as a network printer?

**Comment by michaelrsweet** [comment]
Yes, it should work with a network printer as long as the routing/bridging was setup right. But I'd want to make sure to have as much as possible work from a docker image, to forestall obvious support questions... :)

**Comment by cryptedbv** [comment]
Same here, i'd love to run it in 'Container Station' on a Qnap NAS, i could launch a VM but it would use 200x the resources instead of a simple docker container.
